From 2ab1f66df5712b8f979b244be9b213632d390b4e Mon Sep 17 00:00:00 2001
From: auto commit <unknown@unknown>
Date: Fri, 24 Oct 2008 12:23:12 -0700
Subject: [PATCH] cavium: Add Cavium ethernet as a Kconfig option

Allow the Cavium ethernet to be selected via the normal
kernel configuration process.

Signed-off-by: Tomaso Paoletti <tpaoletti@caviumnetworks.com>
Signed-off-by: Paul Gortmaker <Paul.Gortmaker@windriver.com>
---
 drivers/net/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/net/Kconfig b/drivers/net/Kconfig
index cd5867f..aa89055 100644
--- a/drivers/net/Kconfig
+++ b/drivers/net/Kconfig
@@ -1841,6 +1841,7 @@ config NE_H8300
 	  controller on the Renesas H8/300 processor.
 
 source "drivers/net/fs_enet/Kconfig"
+source "drivers/net/cavium-ethernet/Kconfig"
 
 endif # NET_ETHERNET
 
-- 
1.5.5.1

